flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java

```diff
@@ -33,7 +33,7 @@
 @SuppressWarnings("rawtypes")
 public class MySqlContainer extends JdbcDatabaseContainer {
 
-    public static final String IMAGE = "mysql";
+    public static final String IMAGE = "mysql/mysql-server";
```

Comment:
IIUC, only MySQL-8.0 works in Mac with ARM chips. right?

https://hub.docker.com/r/mysql/mysql-server/

Reply:
No，`mysql/mysql-server:5.7` also can it.  

<img width="1244" alt="image" src="https://user-images.githubusercontent.com/33626973/182029058-fe37da7e-0323-4218-8e13-a394f952ec09.png">

Reply:
https://github.com/ververica/flink-cdc-connectors/blob/b62bb21abfc5bfb83dd181dd84d1a9817686a1ba/flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java#L52

Here, we used the image name mysql/mysql-server:`5.7` or  mysql/mysql-server:`8.0`, so this image can work in Mac with ARM chips.

Reply:
> No，`mysql/mysql-server:5.7` also can it.
> 
> <img alt="image" width="1244" src="https://user-images.githubusercontent.com/33626973/182029058-fe37da7e-0323-4218-8e13-a394f952ec09.png">

`amd64` aka `x86_64` is not same with `arm64`.